pkg/core/interop_neo_test.go

```diff
@@ -457,8 +457,43 @@ func TestAssetGetPrecision(t *testing.T) {
 	require.Equal(t, big.NewInt(int64(assetState.Precision)), precision)
 }
 
+func TestSecp256k1Recover(t *testing.T) {
+	v, context, chain := createVM(t)
+	defer chain.Close()
+
+	v.Estack().PushVal(vm.NewByteArrayItem([]byte{72, 101, 108, 108, 111, 87, 111, 114, 108, 100}))
```

Comment:
If these test vectors are taken from external source, we better provide a link here.
Otherwise explicit signing and recovering will reduce number of magic constants.

Reply:
Sign/recover would require us changing signing routines to return parity and I don't think we need that, so it's perfectly fine with these static vectors, but we need to reference them, true.